<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QMenu" name="menuBluetooth">
     <property name="title">
      <string>Bluetooth</string>
     </property>
     <addaction name="actionConnect"/>
     <addaction name="actionDisconnect"/>
     <addaction name="actionReconnect"/>
    </widget>
    <addaction name="menuBluetooth"/>
   </widget>
   <addaction name="menuConnection"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionReconnect">
   <property name="text">
    <string>Reconnect</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
